schema/me/notifications.js

```diff
@@ -46,6 +47,10 @@ const NotificationsFeedItemType = new GraphQLObjectType({
         return gravity('artworks', { ids: object_ids });
       },
     },
+    image: {
```

Comment:
@mzikherman will this return the image for the artist, or for the first artwork in the notification?